scripts/ee-download.js

```diff
@@ -111,21 +113,28 @@ function camDownloadsWidget(info, holder) {
   function mkServerClickHandler(servers, version, branch) {
     return function (s) {
       var selectedServer = servers[s];
+      var serverInfo = info.servers[selectedServer];
       var release = getReleaseInfo(branch, version);
       var excludesWar = release.excludeFormats && release.excludeFormats.indexOf('war') > -1;
       infoDiv.classList.add('accessible');
 
       runDetails.style.display = 'none';
       fullDetails.style.display = 'inline';
-      
+
       var dl = tmpl('{server}/{branch}/{version}/camunda-bpm-ee-{serverAlias}-{version}-ee', {
         version:  version,
         branch:   (version.indexOf('alpha') > -1) ? 'nightly' : branch,
-        server:   selectedServer,
-        serverAlias: (selectedServer === 'wildfly8')? 'wildfly' : selectedServer
+        server:   serverInfo.path,
+        serverAlias: (selectedServer === 'wildfly8')? 'wildfly' : serverInfo.path
       });
 
-      releaseTitle.innerHTML = version + '-ee for ' + info.servers[selectedServer];
+      releaseTitle.innerHTML = version + '-ee for ' + serverInfo.name;
+      standaloneWebAppTitle.innerHTML = 'Standalone Web Application';
+      standaloneWebAppHint.innerHTML = '';
+      if (version.startsWith('7.19.') && selectedServer === 'wildfly-jakartaee') {
```

Comment:
❓ Will this still work with upcoming versions like 7.20 and the like as well?

Reply:
No, they don't start with 7.19 :)

Reply:
Possibly, yeah, then this would be fine, I guess. Still feels a bit weird with the version number but might be OK 🙂

Reply:
We could have

```suggestion
      if ((version.startsWith('7.19.') || version.startsWith('7.2')) && selectedServer === 'wildfly-jakartaee') {
```

Or is there a comparable version number, let me check...

Reply:
No `7.2.x`, so that would probably work.

Reply:
But what if we reach 7.30.0?!

Reply:
Actually, we could just filter for `wildfly-jakartaee`, I don't think it will be used retrospectively anywhere.
```suggestion
      if (selectedServer === 'wildfly-jakartaee') {
```